### ISSUE TITLE
No module named 'PIL'

### ISSUE THREAD

**Issue by julianriise** [cbh123/narrator#9]
```terminal
python capture.py
Traceback (most recent call last):
  File "/Users/jr/Documents/hobby/narrator/capture.py", line 3, in <module>
    from PIL import Image
ModuleNotFoundError: No module named 'PIL'
```

Not entirely sure what the issue is, wrong Python version? I'm on Python 3.11.4. Tried installing PIL without success:
```terminal
pip3 install PIL
ERROR: Could not find a version that satisfies the requirement PIL (from versions: none)
ERROR: No matching distribution found for PIL
```

**Comment by mattm** [comment]
#2 fixes it

For now:

```bash
pip install Pillow simpleaudio
```

**Comment by rupurt** [comment]
When trying to install those deps on Ubuntu I get the following error

```shell
Building wheels for collected packages: simpleaudio
  Building wheel for simpleaudio (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [37 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib.linux-x86_64-3.10
      creating build/lib.linux-x86_64-3.10/simpleaudio
      copying simpleaudio/__init__.py -> build/lib.linux-x86_64-3.10/simpleaudio
      copying simpleaudio/shiny.py -> build/lib.linux-x86_64-3.10/simpleaudio
      copying simpleaudio/functionchecks.py -> build/lib.linux-x86_64-3.10/simpleaudio
      creating build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      copying simpleaudio/test_audio/c.wav -> build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      copying simpleaudio/test_audio/e.wav -> build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      copying simpleaudio/test_audio/g.wav -> build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      copying simpleaudio/test_audio/left_right.wav -> build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      copying simpleaudio/test_audio/notes_2_16_44.wav -> build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      running build_ext
      building 'simpleaudio._simpleaudio' extension
      creating build/temp.linux-x86_64-3.10
      creating build/temp.linux-x86_64-3.10/c_src
      x86_64-linux-gnu-gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -DDEBUG=0 -I/home/alex/workspace/cbh123/narrator/venv/include -I/usr/include/python3.10 -c c_src/posix_mutex.c -o build/temp.linux-x86_64-3.10/c_src/posix_mutex.o
      x86_64-linux-gnu-gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -DDEBUG=0 -I/home/alex/workspace/cbh123/narrator/venv/include -I/usr/include/python3.10 -c c_src/simpleaudio.c -o build/temp.linux-x86_64-3.10/c_src/simpleaudio.o
      c_src/simpleaudio.c: In function ‘_play_buffer’:
      c_src/simpleaudio.c:164:5: warning: ‘PyEval_InitThreads’ is deprecated [-Wdeprecated-declarations]
        164 |     PyEval_InitThreads();
            |     ^~~~~~~~~~~~~~~~~~
      In file included from /usr/include/python3.10/Python.h:130,
                       from c_src/simpleaudio.h:10,
                       from c_src/simpleaudio.c:7:
      /usr/include/python3.10/ceval.h:122:37: note: declared here
        122 | Py_DEPRECATED(3.9) PyAPI_FUNC(void) PyEval_InitThreads(void);
            |                                     ^~~~~~~~~~~~~~~~~~
      x86_64-linux-gnu-gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -DDEBUG=0 -I/home/alex/workspace/cbh123/narrator/venv/include -I/usr/include/python3.10 -c c_src/simpleaudio_alsa.c -o build/temp.linux-x86_64-3.10/c_src/simpleaudio_alsa.o
      c_src/simpleaudio_alsa.c:8:10: fatal error: alsa/asoundlib.h: No such file or directory
          8 | #include <alsa/asoundlib.h>
            |          ^~~~~~~~~~~~~~~~~~
      compilation terminated.
      error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for simpleaudio
  Running setup.py clean for simpleaudio
Failed to build simpleaudio
ERROR: Could not build wheels for simpleaudio, which is required to install pyproject.toml-based projects
```

**Comment by cbh123** [comment]
Thank you! Just merged #2 which should fix this for Mac
